db/seeds.rb

```diff
@@ -49,76 +49,104 @@
-      retired: false,
-      description: "cool cat 2" }])
-
-Breed.find_by(name: "oncilla").cats.create([
-  { name:"oncilla 1",
-    price: 2000,
-    retired: false,
-    description: "cool cat 3" },
-
-    { name:"oncilla 2",
-      price: 3000,
-      retired: false,
-      description: "cool cat 4" }
-])
-
-Breed.find_by(name: "oriental shorthair").cats.create([
-  { name:"oriental shorthair 1",
-    price: 4000,
-    retired: false,
-    description: "cool cat 5" },
-
-    { name:"oriental shorthair 2",
-      price: 5000,
-      retired: false,
-      description: "cool cat 6" }
-])
-
-Breed.find_by(name: "black cat").cats.create([
-  { name:"black cat 1",
-    price: 3500,
-    retired: false,
-    description: "cool cat 7" },
-
-    { name:"black cat 2",
-      price: 4000,
-      retired: false,
-      description: "cool cat 8" }
-])
-
-Breed.find_by(name: "prionailurus viverrinus").cats.create(
-  { name:"fishing cat 1",
-    price: 1000,
-    retired: false,
-    description: "cool cat 9" })
+
+  def users_minus_dj
+    users_minus_dj = User.all - [User.first]
+  end
+
+  users_minus_dj.each do |user|
+    5.times do |iteration|
+      user.orders.create({ status: [(0..3)].sample })
+    end
+  end
+
+  class BreedMaker
+    attr_accessor :breed_list
+
+    def initialize
+      @breed_list = ["Donskoy", "American Bobtail",
+     "Havanna Brown", "Ocicat",
+     "Oriental Shorthair", "Savannah",
+     "Australian Mist", "British Shorthair",
+     "Cyprus", "Russian Black"]
+    end
+
+    def breed_name
+      @breed_list.delete(@breed_list.sample)
+    end
+
+    def run
+      10.times do |iteration|
+      Breed.create({name: breed_name,
+                    retired: false,
+                    description: Faker::Lorem.sentences(2)})
+      end
+    end
+  end
+
+  BreedMaker.new.run
+
+
+class CatMaker
+  def random_first_name
+    [:first_name, :prefix].sample
+  end
+
+  def random_last_name
+    [:last_name, :first_name, :suffix].sample
+  end
+
+  def cat_first_name
+    Faker::Name.send(random_first_name)
+  end
+
+  def cat_last_name
+    Faker::Name.send(random_last_name)
+  end
+
+  def cat_full_name
+    "#{cat_first_name} #{cat_last_name}"
+  end
+
+  def cents
+    ["00", "50"].sample
+  end
+
+  def dollars
+    common = [10, 12, 15,].sample
+    uncommon = [20, 25, 30, 42].sample
+    rare = [100, 150, 125].sample
+    
+  end
+
+  def run
+    40.times do |iteration|
+      Breed.all.sample.cats.create({
+                name: cat_full_name,
+                price: ((1..50).to_a.sample.to_s + cents).to_i,
+                retired: false,
+                description: Faker::Lorem.sentences(2),
+                image_path: nil
+                })
+    end
+  end
+end
+
+CatMaker.new.run
+
+  def random_cat
+    (1..Cat.count).to_a.sample
+  end
+
+  def random_quantity
+    (1..4).to_a.sample
+  end
+
+  def random_order_cat
+    users_minus_dj.sample.orders.all.sample.order_cats
+  end
+
+  35.times do |iteration|
```

Comment:
Unused block argument - `iteration`. You can omit the argument if you don't care about it.<br>Inconsistent indentation detected.